etna/transforms/encoders/mean_segment_encoder.py

```diff
@@ -74,7 +75,10 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
         df = self.mean_encoder._transform(df)
         segment = segments[0]
         nan_timestamps = df[df.loc[:, self.idx[segment, "target"]].isna()].index
-        df.loc[nan_timestamps, self.idx[:, "segment_mean"]] = [self.global_means[x] for x in segments]
```

Comment:
What was the problem here?

Reply:
Pandas <= 1.2 fails to make this assignment. Some problem with dimensions. To make it explicit, I make a cast here.